lib/vault.go

```diff
@@ -15,17 +15,18 @@ import (
 )
 
 const (
-	STS_DURATION = 3600
+	STS_DURATION_DEFAULT = 3600
 )
 
 var (
 	ErrInvalidCommand = errors.New("Invalid command")
 )
 
 type Vault struct {
-	Vars    map[string]string `json:"vars"`
-	AWSKey  *AWSKey           `json:"aws_key,omitempty"`
-	SSHKeys map[string]string `json:"ssh_keys,omitempty"`
+	Vars     map[string]string `json:"vars"`
+	AWSKey   *AWSKey           `json:"aws_key,omitempty"`
+	SSHKeys  map[string]string `json:"ssh_keys,omitempty"`
+	Duration int64             `json:"duration,omitempty"`
```

Comment:
I wonder if this should be a `time.Duration` instead? Might be a little weird that we store the duration in nanoseconds, but only care about seconds though… but it would make several other operations easier.